### PR TITLE
Point to the new elemental oci images

### PR DESCRIPTION
**Pull request by Itxaka**
Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

File: content/en/docs/Getting started/install.md [hunk 1]
```diff
@@ -106,12 +106,13 @@ For instance, it is possible to install cOS (or any derivative) with the install
 If in the rescue system, or LiveCD you have docker available, it can be used to perform an installation
 
 ```bash
-docker run --privileged -v $DEVICE:$DEVICE -ti quay.io/costoolkit/releases-green:cos-system-0.7.4-2 elemental install --docker-image $IMAGE $DEVICE
+docker run --privileged -v /dev/:/dev/ -ti quay.io/costoolkit/elemental:v0.0.14 install --docker-image $IMAGE $DEVICE
 ```
 
-Where `$IMAGE` is the container image that we want to install (e.g. `quay.io/costoolkit/releases-green:cos-system-0.7.4-2` ), and `$DEVICE` is the the device where to perform the installation to (e.g. /dev/sda).
+Where `$IMAGE` is the container image that we want to install (e.g. `quay.io/costoolkit/releases-green:cos-system-0.7.4-2` ), and `$DEVICE` is the device where to perform the installation to (e.g. /dev/sda).
```
Comment (davidcassany) [inline content/en/docs/Getting started/install.md:112]: I would no use anything below cos-system-0.8 on docs, I think `v0.7.x` does not include elemental

File: content/en/docs/Getting started/install.md [hunk 2]
```diff
@@ -106,12 +106,13 @@ For instance, it is possible to install cOS (or any derivative) with the install
 If in the rescue system, or LiveCD you have docker available, it can be used to perform an installation
 
 ```bash
-docker run --privileged -v $DEVICE:$DEVICE -ti quay.io/costoolkit/releases-green:cos-system-0.7.4-2 elemental install --docker-image $IMAGE $DEVICE
+docker run --privileged -v /dev/:/dev/ -ti quay.io/costoolkit/elemental:v0.0.14 install --docker-image $IMAGE $DEVICE
```
Comment (davidcassany) [inline content/en/docs/Getting started/install.md:109]: just an idea, but probably we can use `quay.io/costoolkit/elemental:latest` here, so there are less chances we have it really outdated after some time.

**Review by davidcassany — APPROVED** [review]
Looks good to me :+1:

Just a couple of nits regarding the image versions, but I am also fine as it is now.

File: content/en/docs/Getting started/install.md (hunk 2 above)
Comment (Itxaka) [reply content/en/docs/Getting started/install.md:109]: I was adamant to include the latest in here...I dont know if it works yet :D

File: content/en/docs/Getting started/install.md (hunk 1 above)
Comment (Itxaka) [reply content/en/docs/Getting started/install.md:112]: Good point, Im gonna point to latest image available instead

**Pull request merged**